code/_onclick/hud/draggable.dm

```diff
@@ -28,10 +28,12 @@
 
 /obj/abstract/screen/fuckbyond/Destroy()
 	..()
+	returnToPool(draggable)
 	draggable = null
 
 /obj/abstract/screen/fuckbyond/MouseUp()
 	returnToPool(draggable)
+	draggable = null
```

Comment:
cluster says all the changes here get handled by `draggable`'s `destroy` and also `resetvariables` and also that it's infinite loop of returntopool